homeassistant/components/switch/__init__.py

```diff
@@ -47,6 +48,7 @@
 PROP_TO_ATTR = {
     'current_power_mwh': ATTR_CURRENT_POWER_MWH,
     'today_power_mw': ATTR_TODAY_MWH,
+    'current_power_watt' : ATTR_CURRENT_POWER_WATT,
```

Comment:
Didn't realize the variables where used in all the other components. Since the device supplies current power usage mwh is not really a useful unit. I clean up the commit and append something a bit more pretty that should pass the CI.